### PR TITLE
Remove lifecycle hook from module

### PR DESCRIPTION
**Pull request by yuvipanda**
This actually doesn't work - am unsure how to make
sure a module doesn't delete a resource.

### CODE REVIEW

**Comment by yuvipanda** [comment]
After this, if you do:

```
terraform init
terraform workspace list
```

You can see the three terraform workspaces containing the installs. `default` is for the two-eye-two-see project, and the other two are for appropriately matching projects.

File: terraform/main.tf [hunk 1]
```diff
@@ -58,10 +58,6 @@ module "gke" {
   # We explicitly set up a core pool, so don't need the default
   remove_default_node_pool   = true
 
-  # NEVER DELETE THE CLUSTER WITHOUT DOUBLE ASKING USER
-  lifecycle {
-    prevent_destroy = true
-  }
```
[the remark this answers is not in the document]
Comment (yuvipanda) [reply terraform/main.tf:64]: It was worse - it was a syntax error! This works in resources and i use it all the time but apparently not in modules. So with this in, you can't actually perform any terraform actions (including `terraform workspace list`)

[the remark this answers is not in the document]
Comment (damianavila) [reply terraform/main.tf:64]: Fun 😛

**Review by damianavila — APPROVED** [review]
LGTM for the little bit I know about terraform 😉 .
Btw, looking quickly online I have not found a way to prevent the deletion in this case.

**Pull request merged**